src/wp-mail.php

```diff
@@ -39,7 +39,7 @@
 $last_checked = get_transient( 'mailserver_last_checked' );
 
 if ( $last_checked ) {
-	wp_die( __( 'Slow down cowboy, no need to check for new mails so often!' ) );
+	wp_die( __( 'Please wait a moment before checking for new emails again.' ) );
```

Comment:
It's known how often email checks can occur so we can show the information.

```suggestion
	wp_die(
		sprintf(
			// translators: %s human readable rate limit.
			__( 'Email checks are rate limited to once every %s.' ),
			human_time_diff( time() - WP_MAIL_INTERVAL, time() )
		),
		__( 'Too Many Requests' ),
		429
	);
```